its/projects/MultiLanguageSupportAngular/ClientApp/package-lock.json

```diff
@@ -3008,21 +3008,15 @@
       "dev": true
     },
     "node_modules/@jridgewell/trace-mapping": {
-      "version": "0.3.18",
-      "resolved": "https://registry.npmjs.org/@jridgewell/trace-mapping/-/trace-mapping-0.3.18.tgz",
-      "integrity": "sha512-w+niJYzMHdd7USdiH2U6869nqhD2nbfZXND5Yp93qIbEmnDNk7PD48o+YchRVpzMU7M6jVCbenTR7PA1FLQ9pA==",
+      "version": "0.3.25",
```

Comment:
This changes the package lock, but where are the changes of the dependency versions?

Reply:
Reading from the internet (this is my very first time updating npm packages with that command), The `package.json` file is not updated
- unless the version ranges specified there need to be changed to accommodate the new versions (aka: package.json specify "myDep": "^5.10.0" and the update is to "5.10.3")
- if it's an indirect dependency we don't have them listed in the package.json

Reply:
Apparently there is also a specific command to update only the package.lock:
 `npm audit fix --package-lock-only`

It means that `npm audit fix --force` should take care of `package.json` if needed.